features/autoZero/d2l-auto-zero-grade.js

```diff
@@ -0,0 +1,50 @@
+import '@brightspace-ui/core/components/tooltip/tooltip.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { HypermediaStateMixin, observableTypes } from '@brightspace-hmc/foundation-engine/framework/lit/HypermediaStateMixin.js';
+import { LocalizeDynamicMixin } from '@brightspace-ui/core/mixins/localize-dynamic-mixin.js';
+
+const rels = {
+	desiredLinkRel: 'self'
+};
+
+class AutoZeroGrade extends LocalizeDynamicMixin(HypermediaStateMixin(LitElement)) {
+	static get properties() {
+		return {
+			isAutoZeroed: { type: Boolean, observable: observableTypes.property, name: 'isAutoZeroed', route: [{observable: observableTypes.property, rel: rels.desiredLinkRel}] },
+		};
+	}
+
+	static get styles() {
+		return css`
+			.auto-zero-tooltip {
+				text-align: center
```

Comment:
nit: 
```suggestion
				text-align: center
```
```suggestion
				text-align: center;
```

(ideally, this would be linted by stylelint)